app/views/user_journey/confirmation.html.erb

```diff
@@ -13,17 +13,10 @@
   <p><%= t 'user_journey.dual_running.not_support_dual_running' %></p>
   <p><%= t 'user_journey.dual_running.how_long_it_takes' %></p>
   <h2 class="govuk-heading-m"><%= t 'user_journey.confirmation.next_steps' %></h2>
-  <h3 class="govuk-heading-s"><%= t 'user_journey.dual_running.if_you_already_added' %></h3>
   <ul class="govuk-list govuk-list--number">
     <li><%= t 'user_journey.dual_running.connection_break' %></li>
     <li><%= t 'user_journey.dual_running.apply_changes' %></li>
   </ul>
-  <h3 class="govuk-heading-s"><%= t 'user_journey.dual_running.if_you_have_not_already_added' %></h3>
```

Comment:
is this due to a rebase or is part of this change?

Reply:
sorry I didn't rebase it properly. Have amended.